src/platform/P6/NetworkCommissioningDriver.h

```diff
@@ -94,6 +94,11 @@ class P6WiFiDriver final : public WiFiDriver
     CHIP_ERROR Init() override;
     CHIP_ERROR Shutdown() override;
 
+    // TODO: Implement this
```

Comment:
please add a github issue number for these TODOs, either one central one with checkboxes or one per platform

Reply:
#15700